load-tiny-infra.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/sh
+
+set -e
+
+cd $(dirname $0)
+python3 core/examples/generated/generate.py /tmp/generated_infras/
+docker cp /tmp/generated_infras/tiny_infra/infra.json osrd-api:/tmp/infra.json
+docker cp api/static/example_rolling_stock.json osrd-api:/tmp/stock.json
+docker exec osrd-api python manage.py import_railjson tiny_infra /tmp/infra.json
+docker exec osrd-api python manage.py import_rolling_stock /tmp/stock.json
+docker exec osrd-editoast editoast generate
```

Comment:
how about making this script generic over which infra to load?

Reply:
I changed the script so that we can specify an infra, but the other ones generate an error